opty/direct_collocation.py

```diff
@@ -133,10 +133,16 @@ class ConstraintCollocator(object):
     """
 
     time_interval_symbol = sym.Symbol('h', real=True)
+    expeirment_data_symbol = sym.Symbol('e', real=True)
```

Comment:
What text editor/IDE do you use? You should find one that checks all this stuff for you automatically. I can give recommendations if you tell me what OS you are working on.